src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/ClientBaseTests.cs

```diff
@@ -174,6 +175,163 @@ public static void ClientBaseOfT_Sync_RoundTrip_Call_Using_NetTcpTransport()
         }
     }
 
+    [Fact]
+    [OuterLoop]
+    public static void DefaultSettings_Echo_RoundTrips_String_StreamedRequest()
+    {
+        string variationDetails = "Client:: BasicHttpBinding/DefaultValues\nServer:: BasicHttpBinding/DefaultValues";
+        string testString = "Hello";
+        StringBuilder errorBuilder = new StringBuilder();
+        bool success = false;
+
+        BasicHttpBinding binding = new BasicHttpBinding(BasicHttpSecurityMode.None);
+        binding.TransferMode = TransferMode.StreamedRequest;
+
+        try
+        {
+            ChannelFactory<IWcfService> factory = new ChannelFactory<IWcfService>(binding, new EndpointAddress(Endpoints.HttpBaseAddress_Basic));
+
+            IWcfService serviceProxy = factory.CreateChannel();
+
+            Stream stream = StringToStream(testString);
+            success = serviceProxy.UploadStream(stream);
+
+            if (!success)
+            {
+                errorBuilder.AppendLine(String.Format("    Error: expected response from service: '{0}' Actual was: '{1}'", true, success));
+            }
+        }
+        catch (Exception ex)
+        {
+            errorBuilder.AppendLine(String.Format("    Error: Unexpected exception was caught while doing the basic echo test for variation...\n'{0}'\nException: {1}", variationDetails, ex.ToString()));
+        }
```

Comment:
Avoid this try/catch format - look at f2fd28a to see how code was changed to avoid the try - catchall pattern. 

We are standardizing on using something like 

```
try {
    // code
} finally {
    if (factory != null && factory.State != CommunicationState.Closed)
    {
        factory.Abort();
    }
}
... 
See dotnet/wcf#168 for details.